timemachine/cpp/src/wrap_kernels.cpp

```diff
@@ -61,6 +62,20 @@ void verify_coords_and_box(
     }
 }
 
+// accumlate_energies handles the combining of energies, summing them up deterministically
+// and returning +inf if there are overflows
+double accumlate_energies(int N, unsigned long long *fixed_u, int overflows) {
```

Comment:
nit: "+inf" -> "nan", "accumlate" -> "accumulate"

Reply:
Fixed in [b82bbf2](https://github.com/proteneer/timemachine/pull/1084/commits/b82bbf2a3cfebcf3604858c3c4d170d37c036bed)